contracts/src/bridge/L2/WrappedERC20.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+
+contract WrappedERC20 is ERC20, AccessControl {
```

Comment:
this Wrapped ERC20 needs to extend the Obscuro ERC20, which in turn extends the ERC20.

The ObscuroERC20 protects the balances